webapp/graphql/groups.js

```diff
@@ -185,6 +185,9 @@ export const groupQuery = (i18n) => {
         }
         avatar {
           url
+          w320: transform(width: 320)
```

Comment:
Maybe it helps to use a GraphQL fragment here so this doesn't happen again (so quickly)?